modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx

```diff
@@ -327,8 +328,7 @@ const PoliciesTable = () => {
 					);
 				})
 			);
-		}
-		else {
+		} else {
```

Comment:
Run afcb to adjust these lines. If possible remove the else.

Reply:
Done.